source/scripting_v3/GTA/Entities/Entity.cs

```diff
@@ -586,6 +586,7 @@ public Vector3 Velocity
 		/// <summary>
 		/// Gets or sets the rotation velocity of this <see cref="Entity"/> in local space.
 		/// </summary>
+		[Obsolete("Entity.RotationVelocity is obsolete because GET_ENTITY_ROTATION_VELOCITY returns the world angular velocity with local to world conversion (without position translation) applied. Use Entity.LocalRotationVelocity instead.")]
```

Comment:
Don't really need to mention that there is no position translation, I'd say that is implicit due to this being a rotation value thus only affected by the rotation matrix.